lib/id3c/cli/command/etl/redcap_det.py

```diff
@@ -100,11 +88,21 @@ def decorator(routine: Callable[..., Optional[dict]]) -> click.Command:
             help        = "Write the output FHIR documents to stdout. You will likely want to redirect this to a file",
             default     = False)
 
+        # XXX FIXME: I don't think @command_for_project should even be
+        # providing the "cache" parameter.  What is cached and where it is stored is
+        # something specific to each REDCap DET routine, not a global invariant.
+        #   -trs, 19 Dec 2019
+        @click.option("--cache-file",
+            metavar = "<cache.pickle>",
+            help = "Local cache file for storing address lookups. Must be specified to cache lookups",
```

Comment:
My 2¢.

```suggestion
            help = "Local file for caching the results of address geocoding. Geocoding lookups will not be cached otherwise.",
```